### DISCUSSION TITLE
Disabling web subscription

### DISCUSSION THREAD

**Discussion by pjlsergeant** [magicbell/community#187] [Questions] [answered]
Hey 👋 

If I've setup a PWA to receive push notifications, I want to disable that when a user logs out of the web-app; the opposite of [subscribe()](https://www.magicbell.com/docs/libraries/webpush#subscribe)

Am I thinking about this wrong? It doesn't seem to be an option.

**Comment by unamashana** [comment]
Hey @pjlsergeant we don't have a way to pause subscriptions, but our new endpoints let you discard a subscription, and you can then add them back again. While they are discarded, we don't deliver to them. Are you logging out in the backend? If so, it'd be easier for you to resubscribe. I can send the curl/openapi snippet if you'd like to try it out.

**Reply by pjlsergeant** [reply, marked as answer]
Hey Hana (@unamashana), yes, that sounds perfect!
